.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## PR Title :boom:
+
+Please title this PR with a summary of the change, along with the JIRA card number.
+
+Suggested formats: 
+
+1. Fixes/Refs #RHIROS-XXX - Title
+2. RHIROS-XXX Title 
```

Comment:
I think we are now using the `RHINENG` project. We can also include that. Otherwise, the rest looks fine.